src/forder.c

```diff
@@ -56,6 +56,15 @@ static uint8_t **key = NULL;
 static int *anso = NULL;
 static bool notFirst=false;
 
+typedef struct {
+    int from;
+    int to;
+    int radix;
+} State;
+
+static State *queue= NULL;
```

Comment:
Or with spaces, just be consistent

```suggestion
static State *queue=NULL;
```